api/src/main/proto/stellarstation/api/v1/monitoring/monitoring.proto

```diff
@@ -231,6 +231,48 @@ message AntennaState {
   antenna.AntennaPolarization polarization = 3;
 }
 
+// The current state of the plan being executed by the ground station.
+message PlanState {
+  // A description of the plan status.
+  enum Status {
+    UNKNOWN = 0;
+
+    // Plan preparation has started. This will be sent only once a short time before AOS when
```

Comment:
Can you avoid specifying how many times a specific state value is sent? It's only monitoring. I want to keep exact timing and exact number of monitoring messages as implementation detail. It is possible that this value is sent twice if sampling rate is high and it is not sent if the rate is low.

Reply:
I think it is also good to avoid ambiguity so I wanted to capture likely user expectations.

For example:

- I'd expect the state messages to be sent soon after the state transition occurs
- states such as COMPLETED and FAILED I'd expect to receive only once. PREPARING is probably open to either interpretation but 'will be sent when the ground station starts preparing for plan execution' implies one time.

We may have a bit of a different view on the priority of monitoring. I think it is not the same as general logging, debug messages etc. It could potentially be used as input to other systems (and be passed back to the stream API) so it should be seen as more than 'best effort'.

Having said all that, I do prefer to avoid having too many hard limits specified in the API. I'll update the text to something I think is still reasonable.